### PR TITLE
docs: enhanced the readme file

### LINKED ISSUE #128: Incomplete command in Readme file
#### Describe the bug
some commands are missing readme file

#### How to Reproduce
Steps to reproduce the issue. Attach all resources that can help us understand the issue:

- Screenshots 
image

#### Expected behavior
command `npm install` must be there

### PR DESCRIPTION
**Pull request by sambhavgupta0705**
**Description**

- Improved the readme file to make it easy for new contributors to locally install the website

**Related issue(s)**

Resolves #128

### CODE REVIEW

**Comment by sambhavgupta0705** [comment]
cc @derberg @AceTheCreator

**Comment by sambhavgupta0705** [comment]
hI, @AceTheCreator @derberg PTAL

File: README.md [hunk 1]
```diff
@@ -2,7 +2,18 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+
+First, fork the repository and clone it.
+
+```bash
+git clone https://github.com/<username>/conference-website.git
+```
```
Comment (AceTheCreator) [inline README.md:10]: Leave a space

**Review by AceTheCreator — COMMENTED** [review]
I left a tiny review

File: README.md (hunk 1 above)
Comment (sambhavgupta0705) [reply README.md:10]: Done

**Comment by sambhavgupta0705** [comment]
Hi Ace ,sorry for late response .I have made the changes now you can review it again

**Review by AceTheCreator — APPROVED** [review]

**Pull request merged**